app/components/reasons_for_rejection_component.html.erb

```diff
@@ -170,6 +170,19 @@
   </div>
 <% end %>
 
+<% if reasons_for_rejection.cannot_sponsor_visa_y_n == 'Yes' %>
+  <div class="app-rejection">
+    <%= content_tag(subheading_tag_name, 'They cannot sponsor your visa', class: 'govuk-heading-s') %>
```

Comment:
Can we use the translation `reasons_for_rejection.cannot_sponsor_visa.title` for the text here?

Reply:
Done, although this isn't how other elements in the template have done things - might make a ticket to change them